Description: Allow building with numexpr 2.8.5

This rejects : for security reasons, and
adds integer ** but with different overflow behaviour

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Bug: https://github.com/pandas-dev/pandas/54546 and 54449
Bug-Debian: https://bugs.debian.org/1049326
Forwarded: partly

--- a/pandas/core/computation/expressions.py
+++ b/pandas/core/computation/expressions.py
@@ -148,8 +148,9 @@ _op_str_mapping = {
     # so sticking with unaccelerated for now GH#36552
     operator.mod: None,
     roperator.rmod: None,
-    operator.pow: "**",
-    roperator.rpow: "**",
+    # disabled due to differing overflow behaviour GH#54546
+    operator.pow: None,
+    roperator.rpow: None,
     operator.eq: "==",
     operator.ne: "!=",
     operator.le: "<=",
--- a/pandas/tests/frame/test_query_eval.py
+++ b/pandas/tests/frame/test_query_eval.py
@@ -1131,7 +1131,7 @@ class TestDataFrameQueryBacktickQuoting:
                 "A (x)": [4, 1, 3],
                 "B(x)": [1, 1, 5],
                 "B (x)": [2, 7, 4],
-                "  &^ :!€$?(} >    <++*''  ": [2, 5, 6],
+                "  &^ !€$?(} >    <++*''  ": [2, 5, 6],
                 "": [10, 11, 1],
                 " A": [4, 7, 9],
                 "  ": [1, 2, 1],
@@ -1224,8 +1224,8 @@ class TestDataFrameQueryBacktickQuoting:
         tm.assert_series_equal(res, expect)
 
     def test_lots_of_operators_string(self, df):
-        res = df.query("`  &^ :!€$?(} >    <++*''  ` > 4")
-        expect = df[df["  &^ :!€$?(} >    <++*''  "] > 4]
+        res = df.query("`  &^ !€$?(} >    <++*''  ` > 4")
+        expect = df[df["  &^ !€$?(} >    <++*''  "] > 4]
         tm.assert_frame_equal(res, expect)
 
     def test_missing_attribute(self, df):
